XenAdmin/Commands/MoveVMCommand.cs

```diff
@@ -67,7 +67,7 @@ public MoveVMCommand(IMainWindow mainWindow, VM vm)
         protected override void ExecuteCore(SelectedItemCollection selection)
         {
 
-            if (selection.AllItemsAre<VM>() && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())
+            if (selection.Count > 1 && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())

```

Comment:
Actually, the count check is not needed, we want the CrossPoolMoveVMCommand even if there is only one VM selected.